packages/dds/matrix/src/test/matrix.applyStashedOp.spec.ts

```diff
@@ -3,7 +3,7 @@
  * Licensed under the MIT License.
  */
 
-import { strict as assert } from "assert";
+import { strict as assert } from "node:assert";
```

Comment:
This was autofixed in all of the test files due to [unicorn/prefer-node-protocol](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/v48.0.1/docs/rules/prefer-node-protocol.md), but the current variation violates [import/no-nodejs-modules](https://github.com/import-js/eslint-plugin-import/blob/v2.29.0/docs/rules/no-nodejs-modules.md). Not sure which error is better to supress, or if there's a way around it.

Reply:
You should be able to update the package's eslintrc to fix this. It should already have an `no-nodejs-modules` override that specifies `assert` - you'll just want to update that to say `node:assert` instead.